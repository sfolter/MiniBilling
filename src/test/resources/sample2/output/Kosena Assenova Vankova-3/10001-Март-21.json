{
    "documentDate": "2022-08-19T13:16:32.036326400",
    "documentNumber": "10001",
    "consumer": "Kosena Assenova Vankova",
    "reference": "3",
    "totalAmount": "335.72 EUR",
    "totalAmountWithVat": "519.99 EUR",
    "lines": [
        {
            "index": 1,
            "quantity": 392.4,
            "lineStart": "2021-01-01T14:40:00Z",
            "lineEnd": "2021-02-28T23:59:59Z",
            "product": "gas",
            "price": 0.7,
            "priceList": 1,
            "amount": "274.68 EUR"
        },
        {
            "index": 2,
            "quantity": 43.6,
            "lineStart": "2021-03-01T00:00:00Z",
            "lineEnd": "2021-03-11T07:00:00Z",
            "product": "gas",
            "price": 1.4,
            "priceList": 1,
            "amount": "61.04 EUR"
        }
    ],
    "taxes": [
        {
            "index": 1,
            "lines": 1,
            "name": "Standing charge",
            "quantity": 58,
            "unit": "days",
            "price": 1.6,
            "amount": "92.80 EUR"
        },
        {
            "index": 2,
            "lines": 2,
            "name": "Standing charge",
            "quantity": 10,
            "unit": "days",
            "price": 1.6,
            "amount": "16.00 EUR"
        }
    ],
    "vat": [
        {
            "index": 1,
            "lines": 1,
            "taxes": 0,
            "taxedAmountPercentage": 60,
            "percentage": 20,
            "amount": "32.96 EUR"
        },
        {
            "index": 2,
            "lines": 1,
            "taxes": 0,
            "taxedAmountPercentage": 40,
            "percentage": 10,
            "amount": "10.99 EUR"
        },
        {
            "index": 3,
            "lines": 0,
            "taxes": 1,
            "taxedAmountPercentage": 100,
            "percentage": 20,
            "amount": "18.56 EUR"
        },
        {
            "index": 1,
            "lines": 2,
            "taxes": 0,
            "taxedAmountPercentage": 60,
            "percentage": 20,
            "amount": "7.32 EUR"
        },
        {
            "index": 2,
            "lines": 2,
            "taxes": 0,
            "taxedAmountPercentage": 40,
            "percentage": 10,
            "amount": "2.44 EUR"
        },
        {
            "index": 3,
            "lines": 0,
            "taxes": 2,
            "taxedAmountPercentage": 100,
            "percentage": 20,
            "amount": "3.20 EUR"
        }
    ]
}