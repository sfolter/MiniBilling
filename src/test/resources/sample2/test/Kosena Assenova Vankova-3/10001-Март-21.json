{
    "documentDate": "2022-08-29T11:50:37.501053200",
    "documentNumber": "10001",
    "consumer": "Kosena Assenova Vankova",
    "reference": "3",
    "totalAmount": "335.72 BGN",
    "totalAmountWithVat": "922.86 BGN",
    "lines": [
        {
            "index": 1,
            "quantity": 392.4,
            "lineStart": "2021-01-01T14:40:00Z",
            "lineEnd": "2021-02-28T23:59:59Z",
            "product": "gas",
            "price": 0.7,
            "priceList": 1,
            "amount": "274.68 BGN"
        },
        {
            "index": 2,
            "quantity": 43.6,
            "lineStart": "2021-03-01T00:00:00Z",
            "lineEnd": "2021-03-11T07:00:00Z",
            "product": "gas",
            "price": 1.4,
            "priceList": 1,
            "amount": "61.04 BGN"
        }
    ],
    "taxes": [
        {
            "index": 1,
            "lines": 1,
            "name": "Standing charge",
            "quantity": 58,
            "unit": "days",
            "price": 1.6,
            "amount": "92.80 BGN"
        },
        {
            "index": 2,
            "lines": 2,
            "name": "Standing charge",
            "quantity": 10,
            "unit": "days",
            "price": 1.6,
            "amount": "16.00 BGN"
        }
    ],
    "vat": [
        {
            "index": 1,
            "lines": 1,
            "percentage": 20,
            "amount": "32.96 BGN"
        },
        {
            "index": 2,
            "lines": 1,
            "percentage": 10,
            "amount": "10.99 BGN"
        }
    ]
}