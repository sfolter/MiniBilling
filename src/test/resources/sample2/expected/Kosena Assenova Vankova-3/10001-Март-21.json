{
  "documentNumber": "10002",
  "consumer": "Kosena Assenova Vankova",
  "reference": "3",
  "totalAmount": 60.8,
  "lines": [
    {
      "index": 1,
      "quantity": 106.7,
      "lineStart": "2021-01-01Ð¢14:40:00Z",
      "lineEnd": "2021-02-28T21:59:59Z",
      "product": "gas",
      "price": 0.4,
      "priceList": 1,
      "amount": 74.69
    },{
      "index": 2,
      "quantity": 45.3,
      "lineStart": "2021-02-28T22:00:00Z",
      "lineEnd": "2021-03-25T07:11:00Z",
      "product": "gas",
      "price": 0.4,
      "priceList": 1,
      "amount": 18.12
    }
  ]
}